Fix:
error: no viable conversion from returned value of type 'basic_string<std::__1::basic_stringstream<wchar_t, std::__1::char_traits<wchar_t>, std::__1::allocator<wchar_t> >::char_type, char_traits<wchar_t>, allocator<wchar_t>>' to function return type 'basic_string<char, char_traits<char>, allocator<char>>'
https://github.com/dirkvdb/lastfmlib/pull/13
--- lastfmlib/utils/stringoperations.h.orig
+++ lastfmlib/utils/stringoperations.h
@@ -55,7 +55,7 @@ namespace StringOperations
     }
 
     template<typename T>
-    inline std::string toWstring(T& numeric)
+    inline std::wstring toWstring(T& numeric)
     {
         std::wstringstream ss;
         ss << numeric;
